### PR TITLE
HTML5 based login page, REVISION 2

### PR DESCRIPTION
**Pull request by bradymiller**
html5 login and new logo from Robert's work
(note did not touch the tinylogo formatting to ensure don't break ongoing use of them)

Can be demo'd here:
http://www.open-emr.org/wiki/index.php/Development_Demo#192.168.1.131

Note the first revision of this PR can be found here:
https://github.com/openemr/openemr/pull/298

### CODE REVIEW

**Comment by bradymiller** [comment]
Note the first revision of this PR can be found here:
https://github.com/openemr/openemr/pull/298

File: interface/login/login.php [hunk 1]
```diff
@@ -29,267 +29,268 @@
...
+                        if (count($result) == 1) {
+	                          $resvalue = $result[0]{"name"};
+	                          echo "<input type='hidden' name='authProvider' value='" . attr($resvalue) . "' />\n";
+                        }
+                        // collect default language id
+                        $res2 = sqlStatement("select * from lang_languages where lang_description = ?",array($GLOBALS['language_default']));
+                        for ($iter = 0;$row = sqlFetchArray($res2);$iter++)
+                            $result2[$iter] = $row;
+                        if (count($result2) == 1) {
+                            $defaultLangID = $result2[0]{"lang_id"};
```
Comment (Wakie87) [inline interface/login/login.php]: @bradymiller Why is this not $result2[0]["lang_id"];?

**Comment by bradymiller** [comment]
Couple issues noted:
1. Main logo via SVG doesn't seem very crisp. Anybody else notice this on testing?
2. Should the Language selector be below the user/pass input fields or is it ok to have it above them?

File: interface/login/login.php (hunk 1 above)
Comment (bradymiller) [inline interface/login/login.php]: btw, I didn't change this code in this PR.
Not sure why it's using those brackets.

Comment (Wakie87) [inline interface/login/login.php]: Yeah i had noticed it when playing around with pdo. It appears a few time in the login.php page.

**Comment by bradymiller** [comment]
btw,
The final author of this code will be Robert Down.

**Comment by bradymiller** [comment]
This is ready for prime time testing and review. Can be demo'd here:
http://www.open-emr.org/wiki/index.php/Development_Demo#192.168.1.131

**Comment by bradymiller** [comment]
Just brought this into the codebase.

**Pull request closed without merging**